src/common.c

```diff
@@ -185,6 +185,9 @@ int ofi_ep_bind_valid(struct fi_provider *prov, struct fid *bfid, uint64_t flags
 			return -FI_EBADFLAGS;
 		}
 		break;
+	case FI_CLASS_SRX_CTX:
+		/* Flags can be ignored */
+		break;
```

Comment:
Since flags are not valid for shared contexts, they shouldn't be passed while binding a shared context to an EP. This switch case can be dropped altogether.
@j-xiong may be we should clarify this in the man pages?